browser/rewards_data/counters/rewards_data_counter_utils.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/rewards_data/counters/rewards_data_counter_utils.h"
+
+#include "base/strings/utf_string_conversions.h"
+#include "brave/browser/rewards_data/counters/rewards_counter.h"
+#include "chrome/browser/profiles/profile.h"
+#include "brave/common/pref_names.h"
+#include "chrome/grit/generated_resources.h"
+#include "brave/components/rewards_data/core/pref_names.h"
+#include "components/prefs/pref_service.h"
+#include "components/strings/grit/components_strings.h"
+#include "ui/base/l10n/l10n_util.h"
+#include "ui/base/text/bytes_formatting.h"
+
+namespace rewards_data_counter_utils {
+
+base::string16 GetBraveCounterTextFromResult(
```

Comment:
these methods seem out of place here, shouldn't they just be anonymous methods in the UI class?